### ISSUE TITLE
Kernel dying mac os Mojave 10.14

### ISSUE THREAD

**Issue by efago** [Microsoft/LightGBM#1976]
After grappling with installation of LightGBM, which I couldn't get an easy installation with anaconda, I couldn't run lightgbm on jupyter because it is crashing instantly. Please give me your feedback, I am stuck. :(

## Environment info

Operating System: Mac Mojave 14

CPU/GPU model: 16 GB 2400 MHz DDR4, 2.2 GHz Intel Core i7

C++/Python/R version: python 3.7

LightGBM version or commit hash: 2.2

## Error message
The kernel appears to have died. It will restart automatically.

## Steps to reproduce

param = {'num_leaves': 111,
         'min_data_in_leaf': 149, 
         'objective':'regression',
         'max_depth': 9,
         'learning_rate': 0.005,
         "boosting": "gbdt",
         "feature_fraction": 0.7522,
         "bagging_freq": 1,
         "bagging_fraction": 0.7083 ,
         "bagging_seed": 11,
         "metric": 'rmse',
         "lambda_l1": 0.2634,
         "random_state": 133,
         "verbosity": -1}
folds = KFold(n_splits=5, shuffle=True, random_state=15)
oof = np.zeros(len(train))

for fold_, (trn_idx, val_idx) in enumerate(folds.split(train.values, target.values)):
    train_mat = lgb.Dataset(train.iloc[trn_idx][features],
                           label=target.iloc[trn_idx],
                          )
    val_mat = lgb.Dataset(train.iloc[val_idx][features],
                           label=target.iloc[val_idx],
                           categorical_feature=categorical_feats
                          )

    num_round = 10000
    clf = lgb.train(param,
                    trn_data,
                    num_round,
                    valid_sets = [train_mat, val_mat],
                    verbose_eval=100,
                    early_stopping_rounds = 200)
    
    oof[val_idx] = clf.predict(train.iloc[val_idx][features], num_iteration=clf.best_iteration)
    
    feature_importance = clf.feature_importance()

**Comment by drabrucelee** [comment]
I am also stuck at that point using the predict, hopping for feedback!

**Comment by efago** [comment]
I hope someone with a better knowledge would help.

**Comment by StrikerRUS** [comment]
@efago @drabrucelee What about simple `python` command? Does it crashes too?

Please post a log from the Jupyter console. You haven't share a [MCVE](https://stackoverflow.com/help/mcve), so without any logs it hard to say what's going wrong here.

**Comment by efago** [comment]
@StrikerRUS I have fixed the problem. During the installation of lightgbm, I was having problems and I installed and uninstalled lightgbm a couple of times without taking care of openmp. So I ended up with multiple openmp packages in my system and it was the cause of the crash. Thanks for your help.

**Comment by drabrucelee** [comment]
Thanks a lot, the issue is fixed.

**Comment by deepdalytics** [comment]
@efago I seem to have the same problem. How did you remove the additional openmp packages/clean up afterwards? Thank you
